comms-templates/distributors-announcement-email.md

```diff
@@ -0,0 +1,60 @@
+_Use this email template for pre-disclosing security vulnerabilities to distributors-announce._
+
+TO: `distributors-announce@kubernetes.io`
+
+SUBJECT: `[EMBARGOED] $CVE: $SUMMARY`
+
+---
+
+### EMBARGOED
+
+The information contained in this email is **[under
+embargo](https://github.com/kubernetes/security/blob/master/private-distributors-list.md#embargo-policy)**
```

Comment:
Should we change the format of this to not be a markdown link to make copy/paste of the template easier?

Reply:
I usually copy the rendered markdown. We should probably decide whether we want to optimize the template for plain-text though. The precedent is to send formatted emails, both for distributors & public announcements, and I haven't heard any complaints.

Reply:
That makes sense - copy/pasting the rendered markdown and sending formatted emails sounds good to me. Thank you!